.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,47 @@
+name: "\U0001F41B Bug Report"
+description: Report a bug
+title: "Bug: TITLE"
+labels: ["bug"]
+body:
+  - type: textarea
+    id: describe_behaviour
+    attributes:
+      label: Describe Behaviour
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected_behaviour
+    attributes:
+      label: Expected Behaviour
+    validations:
+      required: true
+
+  - type: textarea
+    id: current_behaviour
+    attributes:
+      label: Current Behaviour
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduction_steps
+    attributes:
+      label: Reproduction Steps
+    validations:
+      required: true
+
+  - type: textarea
+    id: code_snippet
+    attributes:
+      label: Code Snippet
+    validations:
+      required: false
+
+  - type: textarea
+    id: possible_solution
+    attributes:
+      label: Possible Solution
+    validations:
+      required: false
+
```

Comment:
Not sure how to best fit this in, but for bugs can we add something around:
1. the version of our software/its deps
2. and maybe the tricky one, version of external application for integrations?